src/code-completion/assistant-completion.ts

```diff
@@ -195,6 +207,28 @@ export async function providesCodeCompletion(
     }
 
     snippetCompletion.insertText = new vscode.SnippetString(vscodeFormatCode);
+    snippetCompletion.command = {
```

Comment:
This is why the tests fail: we overrid the command from like 185. The fact we save the usage of the recipe should be added in the old command (e.g. `codiga.registerUsage`).

Warning: in `codiga.registerUsage`, we need to make sure the recipe has a shortcut before we save it.